### PR TITLE
master-qa-50750

### PR DESCRIPTION
**Pull request by jaydawu**
https://issues.liferay.com/browse/LRQA-50750
Submit for master.

### CODE REVIEW

**Comment by jaydawu** [comment]
ci:test:sf

**Comment by Sharryxue** [comment]
Just starting review.

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/gdpr/ExportPersonalData.path [hunk 1]
```diff
@@ -25,6 +25,11 @@
 	<td>//table//tr[contains(.,'${key_application}')]//input[@type='checkbox']</td>
 	<td></td>
 </tr>
+<tr>
+	<td>APPLICATION_TABLE_HEAD_ITEM</td>
```
Comment (Sharryxue) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/gdpr/ExportPersonalData.path]: Would you please double check if the exist locator _LexiconTable#TABLE_HEADER_ works? Thanks in advance.

**Review by Sharryxue — CHANGES_REQUESTED** [review]
Hello @jaydawu 
Thanks for working on the test, would you please double check if there is exist locator can be reused instead of creating the new one?

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/gdpr/ExportPersonalData.path (hunk 1 above)
Comment (jaydawu) [reply portal-web/test/functional/com/liferay/portalweb/paths/portlet/gdpr/ExportPersonalData.path]: Sure. Thank you!

**Comment by jaydawu** [comment]
Re-pushed to https://github.com/Sharryxue/liferay-portal/pull/403

**Pull request closed without merging**